keylime-ansible-gcp/README.md

```diff
@@ -0,0 +1,37 @@
+# Ansible Keylime for Google Cloud
+Ansible role to deploy [Keylime](https://github.com/keylime/keylime) with the [rust agent](https://github.com/keylime/rust-keylime) against a Virtualized TPM.
+
+This role is currently configured to work with a Fedora 35 instance on Google Cloud with the vTPM enabled. 
+
+Contributions are welcome, should anyone wish to have this role provision other Linux distributions.
+
+For details on using Keylime, please consult the
+[project documentation](https://keylime-docs.readthedocs.io/en/latest/)
+
+For details on the Rust agent, please consult the [repository](https://github.com/keylime/rust-keylime).
+
+## Usage
+Run the playbook against your target remote host(s). Note: the hosts must have the vTPM enabled in the Google Cloud Console. 
```

Comment:
A cooler next step would be to have this playbook actually create the host(s) in GCP with all of the right settings (OS, image, vTPM, etc). It could have some parameters passed on the cmd line to override some defaults (like compute size, storage size, etc). And since almost all real world scenarios don't have the agent and the server side components (verifier, registrar, etc) running on the same host, it could create a couple of hosts, say 1 where the agent is running and 1 where the server side components are running.

I've done these kinds of playbooks before in ansible for AWS, so I'm making an assumption that they can also work for GCP.